apps/music/test/marionette/lib/statusbar.js

```diff
@@ -20,9 +20,10 @@ Statusbar.prototype = {
   },
 
   waitForPlayingIndicatorShown: function(shouldBeShown) {
-    this.client.waitFor(function() {
-      var shown = this.playingIndicator.displayed();
-      return shown === shouldBeShown;
-    }.bind(this));
+    if (shouldBeShown) {
+      this.client.helper.waitForElement(this.playingIndicator);
+    } else {
+      this.client.helper.waitForElementToDisappear(this.playingIndicator);
```

Comment:
Is that a new call? I don't remember seeing this when I reviewed / reworked the tests.